bucket/sublime-text.json

```diff
@@ -50,19 +32,32 @@
             ]
         }
     },
+    "pre_install": [
+        "$file = \"$dir\\sublime-text-install-context.reg\"",
+        "if (Test-Path $file) {",
+        "    $sublimepath = \"$dir\\sublime_text.exe\".Replace('\\', '\\\\')",
+        "    $content = Get-Content $file",
+        "    $content = $content.Replace('$sublime', $sublimepath)",
+        "    if ($global) { $content = $content.Replace('HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE') }",
+        "    Set-Content $file $content -Encoding Ascii -Force",
+        "}",
+        "$file = \"$dir\\sublime-text-uninstall-context.reg\"",
+        "if (Test-Path $file) {",
+        "    $content = Get-Content $file",
+        "    if ($global) { $content = $content.Replace('HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE') }",
+        "    Set-Content $file $content -Encoding Ascii -Force",
+        "}"
+    ],
+    "notes": [
+        "Sublime Text may be downloaded and evaluated for free, however a license must be purchased for continued use.",
+        "There is currently no enforced time limit for the evaluation.",
+        "For more information please see 'https://www.sublimetext.com/3'.",
+        "Add Sublime Text as a context menu option by running: \"reg import '$dir\\sublime-text-install-context.reg'\""
+    ],
     "bin": "subl.exe",
+    "shortcuts": [["sublime_text.exe", "Sublime Text 3"]],
```

Comment:
Format manifest using `formatjson`, `checkver` or just with vscode

Reply:
sorry for that,  that the the beavior of my default vscode format plugin. I changed it back to vscode default and reformated it now.